modules/sitemaps/sitemap-builder.php

```diff
@@ -1319,6 +1319,20 @@ private function video_post_to_sitemap_item( $post ) {
 
 		/** This filter is already documented in core/wp-includes/feed.php */
 		$content = apply_filters( 'the_content_feed', $content, 'rss2' );
+		
+		// Include thumbnails for VideoPress videos, use blank image for others
+		if ( 'complete' === get_post_meta( $post->ID, 'videopress_status', true ) && has_post_thumbnail( $post ) ) {
+			$video_thumbnail_url = get_the_post_thumbnail_url( $post );
+		} else {
+			/**
+			 * Filter the thumbnail image used in the video sitemap for non-VideoPress videos.
+			 *
+			 * @since 7.2.0
+			 *
+			 * @param string $str Image URL.
+			 */
+			$video_thumbnail_url = apply_filters( 'jetpack_video_sitemap_default_thumbnail', 'https://s0.wp.com/i/blank.jpg' );
```

Comment:
Any thought on using the video default image that ships with core, e.g. https://kraftbj.wpsandbox.me/wp-includes/images/media/video.png

Any pros or cons from your POV?

Reply:
The core image is better than the blank one, as far as imparting the nature of the file goes (i.e. that it's a video).

However, Google suggests a minimum thumbnail size of 160x90:

https://support.google.com/webmasters/answer/156442#thumbnails

... which makes the core image too small.

I did not actually test to see if Search Console would complain about a too-small image though.

Reply:
I would be fine assuming it would complain. So, let's roll with blank for now and we can accept a future PR to provide a better default image.